src/main/io/serial_cli.c

```diff
@@ -1214,24 +1210,27 @@ static void cliRxFail(char *cmdline)
     }
 }
 
-static void printAux(uint8_t dumpMask, const modeActivationProfile_t *defaultModeActivationProfile)
+static void printAux(uint8_t dumpMask, const modeActivationProfile_t *modeActivationProfile, const modeActivationProfile_t *defaultModeActivationProfile)
 {
+    const char *format = "aux %u %u %u %u %u\r\n";
     // print out aux channel settings
     for (uint32_t i = 0; i < MAX_MODE_ACTIVATION_CONDITION_COUNT; i++) {
-        const modeActivationCondition_t *mac = &modeActivationProfile()->modeActivationConditions[i];
-        const modeActivationCondition_t *macDefault = &defaultModeActivationProfile->modeActivationConditions[i];
-        const bool equalsDefault = mac->modeId == macDefault->modeId
-            && mac->auxChannelIndex == macDefault->auxChannelIndex
-            && mac->range.startStep == macDefault->range.startStep
-            && mac->range.endStep == macDefault->range.endStep;
-        const char *format = "aux %u %u %u %u %u\r\n";
-        cliDefaultPrintf(dumpMask, equalsDefault, format,
-            i,
-            macDefault->modeId,
-            macDefault->auxChannelIndex,
-            MODE_STEP_TO_CHANNEL_VALUE(macDefault->range.startStep),
-            MODE_STEP_TO_CHANNEL_VALUE(macDefault->range.endStep)
-        );
+        const modeActivationCondition_t *mac = &modeActivationProfile->modeActivationConditions[i];
+        bool equalsDefault = true;
```

Comment:
@martinbudden To me the change looks good. I think the initial value of `equalsDefault` is irrelevant, as it should never get used when there is no default config. When I wrote this, I tried a number of different ways to express the conditions for `equalsDefault` and the flags in `dumpMask`, but this was the best one I could come up with.